dashboards/sustainability-proxy-metrics/sustainability-proxy-metrics.yaml

```diff
@@ -67,10 +67,13 @@ datasets:
               Type: INTEGER
             - Name: usage_quantity
               Type: DECIMAL
+              SubType: FIXED
```

Comment:
To my experience SubType are causing issues

Reply:
Wasn't an explicit choice of mine. That's how the latest export looks like. Fails on import when you don't have a recent boto3 version. Highlighted the [required version](https://github.com/boto/boto3/blob/develop/CHANGELOG.rst#12877) in the commit comment.